test/unit/mdc-drawer/persistent.foundation.test.js

```diff
@@ -51,11 +51,9 @@ test('defaultAdapter returns a complete adapter implementation', () => {
 });
 
 test('#init is super.init', () => {
```

Comment:
I think we can eliminate this test entirely. I don't think its doing anything at this point.

Reply:
Took it out and brought branch up to date with the latest mdc version. Not sure what to do about the CLA signing.